nipype/interfaces/fsl/epi.py

```diff
@@ -264,15 +263,14 @@ def _list_outputs(self):
                 base_path = None
         else:
             base = split_filename(self.inputs.in_file)[1] + '_base'
-        outputs['out_fieldcoef'] = self._gen_fname(base, suffix='_fieldcoef',
+        self.outputs.out_fieldcoef = self._gen_fname(base, suffix='_fieldcoef',
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unexpected keyword argument 'cwd' in method call](https://www.codacy.com/app/krzysztof-gorgolewski/nipy_nipype/file/2065943999/issues/source?bid=2767207&fileBranchId=2890884#l266)